src/Stoa.ts

```diff
@@ -0,0 +1,20 @@
+import express from "express";
+
+class Stoa {
+    public stoa: express.Application;
+
+    constructor () {
+        this.stoa = express();
+
+        this.stoa.get("/validators",
+            (req: express.Request, res: express.Response, next: express.NextFunction) => {
+            res.send(req.query.height);
```

Comment:
Where does the `height` property come from? Is the support code for this automatically generated?

Also, height should be an optional parameter. Does this code handle that well?

Reply:
In the get method request, the query parameter is stored in a dynamic object and this is an option.
IF processing will be done in logic construction in the future.

Reply:
Alright!